test/elm/comparison/data.coffee

```diff
@@ -6117,8 +6283,50 @@ define TupleAndNullList: {a: '1'} ~ {null}
 define NullListAndTuple: {null} ~ {a: '1'}
 ###
 
+###
+Translation Error(s):
+[6:21, 6:31] Call to operator Equivalent(System.Any,System.Any) is ambiguous with: 
+  - Equivalent(System.Decimal,System.Decimal)
+  - Equivalent(System.Boolean,System.Boolean)
+  - Equivalent(list<System.Integer>,list<System.Integer>)
+  - Equivalent(System.Integer,System.Integer)
+  - Equivalent(System.DateTime,System.DateTime)
+  - Equivalent(System.Ratio,System.Ratio)
+  - Equivalent(System.Date,System.Date)
+  - Equivalent(System.String,System.String)
+  - Equivalent(System.Quantity,System.Quantity)
+  - Equivalent(System.Time,System.Time)
+###
```

Comment:
The new CQL-to-ELM is choking on this expression:
```
define ANull_BNull: null ~ null
```

I guess that's fair since it can't tell the type of `null` -- and no one would ever code this expression _literally_ this way anyway.

Still, we want to test `null ~ null`.  I wonder if we can trick it by rewriting the test to:

```
define ANull_BNull: (null as String) ~ (null as String)
```

Can you try?  In theory we should actually try that for every type listed in the error above, but really... that's kind of craziness.  I'll let you decide.